src/MIDebugEngine/AD7.Impl/AD7Engine.cs

```diff
@@ -359,6 +361,21 @@ public int CreatePendingBreakpoint(IDebugBreakpointRequest2 pBPRequest, out IDeb
             return Constants.S_OK;
         }
 
+        public int GetBPLongBindTimeout()
+        {
+            if (s_bpLongBindTimeout == 0)
+            {
+                s_bpLongBindTimeout = 250; // default is to wait a quarter of a second
+
+                object timeoutExtension = _configStore.GetEngineMetric("BpLongBindTimeoutExtension");
+                if (timeoutExtension != null && timeoutExtension is int && ((int)timeoutExtension == 1))
+                {
+                    s_bpLongBindTimeout = 2500; // if its set, make it 10x
```

Comment:
You might want to make it longer than this so you can debug the test without issue